class Failure{
  final String message;
  const Failure([this.message = 'An unexpected error ocurred']);
}